package.json

```diff
@@ -8,11 +8,12 @@
   },
   "scripts": {
     "fix": "npm run lint:es -- --fix && npm run lint:css -- --fix",
+    "htmlhint": "htmlhint \"**/*.html\" -i \"common/**/*.html\" -f unix",
     "lint": "npm run lint:es && npm run lint:css && npm run vnu-jar && npm run lint:spelling",
```

Comment:
this should call `npm run lint:html` instead of `npm run vnu-jar`!

Reply:
Good call! fixed